frontend/components/Vehicles/ConfirmDialog.vue

```diff
@@ -0,0 +1,37 @@
+<template>
+  <v-dialog v-model="showDialog" max-width="500px">
+    <v-card>
+      <v-card-title>
+        <slot></slot>
+      </v-card-title>
+
+      <v-card-actions>
+        <v-spacer></v-spacer>
+        <v-btn color="blue darken-1" text @click="close">
+          Cancel
+        </v-btn>
+        <v-btn color="blue darken-1" text @click="confirm">
+          Confirm
+        </v-btn>
+      </v-card-actions>
+    </v-card>
+  </v-dialog>
+</template>
+
+<script>
+export default {
+  name: 'ConfirmDialog',
+  props: ["showDialog"],
```

Comment:
You should write props like this:
```suggestion
  props: {
    show: { type: Boolean, required: true }
   },
```
Now you have an explicit type and it's required as well.